code/modules/awaymissions/mission_code/ruins/moonoutpost19.dm

```diff
@@ -0,0 +1,37 @@
+
+/obj/item/paper/fluff/ruins/moonoutpost19/entrance
+	name = "hastily written note"
+	info = "A warning to whose reading this, things here went really wrong. Way too damn wrong. Turn back while you can, there's nothing left for anyone. Not anymore."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/rockstarcop
+	name = "eerie note"
+	info = "(you see some unintelligible scribbles covered in blood, only words you can choose are...) <br><br>If everybody knew, I never knew <br>I will wake up in a new life<br> Down by the seaside..."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/engineering
+	name = "emergency procedures"
+	info = "(most of the parts are unreadable due to stains on the paper, the only words you can make out is...)<br><br> ...incase of power outages, use the provided portable generators. It is recommended that setting up two of them at once rather than working with one generator on high power levels. It is also recommended using the pre-marked spots for generator setup. If you are uncertain where it is, use a t-ray scanner to check the floors. Provided plasma sheets will suffice until maintai..."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/rdnote
+	name = "Personal Log"
+	info = "I am telling you, we're wasting our resources here with unnecessary things while we have means to make absolute difference. I have discussed this matter with Operations Officer, tomorrow we'll be expecting a shipment of three alien specimen eggs. They expect us to do something 'useful' about them, heh. I have made it this far, i sure can climb this ladder even further."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/boreraccountant
+	name = "confusing note"
+	info = "I had a weird dream recently about something small catching me off-guard, from under my desk where they've hidden. <br><br>And now something in my mind whispers to me about 'tax norms', how it is implemented and various terms i have never heard before. <br><br> A note to self: don't push yourself too hard over small handicaps. It obviously makes you go insane, in a scary way too... And ask your therapist for extra session regarding to this matter."
+
+/obj/item/paper/fluff/ruins/moonoutpost19/log1
+	name = "Chamber #1 Log"
+	info = "<hr><b>Data Log #82<br><br> Subject: Yes<br><br> Details:</b><br><br> I fricking hate our so called 'Senior Researcher' Mr. Wood. Someone has to show him how we handle the pricks whose nose in the air in this line of business.<br><br> Therefore i have developed a gas cannon. What this device does is simply compressing air and releasing it in a specific direction -in this case it would be the airlock of this very chamber- at a funny velocity. He checks the containtment chambers daily, all i have to do is keep this beauty locked and loaded until he opens the airlock. Afterall the only difference between messing around and science is writing it down. I bet you won't be so senior about this."
```

Comment:
```suggestion
	info = "<hr><b>Data Log #82<br><br> Subject: Yes<br><br> Details:</b><br><br> I fricking hate our so called 'Senior Researcher' Mr. Wood. Someone has to show him how we handle the pricks whose nose in the air in this line of business.<br><br> Therefore i have developed a gas cannon. What this device does is simply compressing air and releasing it in a specific direction -in this case it would be the airlock of this very chamber- at a funny velocity. He checks the containment chambers daily, all i have to do is keep this beauty locked and loaded until he opens the airlock. After all the only difference between messing around and science is writing it down. I bet you won't be so senior about this."
```